compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenAttrs.cpp

```diff
@@ -460,6 +462,23 @@ int64_t WorkgroupMappingAttr::getRelativeIndex() const {
   return getMappingId();
 }
 
+//===---------------------------------------------------------------------===//
+// iree_codegen.encoding_layout
+//===---------------------------------------------------------------------===//
+
+MaterializeEncodingInfo
+EncodingLayoutAttr::getEncodingInfo(RankedTensorType type) const {
+  return MaterializeEncodingInfo{};
+}
+
+SmallVector<Operation *>
+EncodingLayoutAttr::lowerComputeOp(OpBuilder &b, Operation *op,
+                                   TypeRange convertedResTypes,
+                                   ValueRange convertedOperands) const {
+  Operation *clonedOp = clone(b, op, convertedResTypes, convertedOperands);
+  return SmallVector<Operation *>{clonedOp};
```

Comment:
Will this ever return multiple ops, and is that why it returns a vector? If so, how does the caller know what to make of the multiple ops that are returned? I can see the call site introduced in this commit, takes only the last element in the vector (`.back()`). If there will eventually be multiple ops, how does this caller know that `.back()` is the element that it wants ?

Reply:
I made it return `Operation *` in a following commit: https://github.com/iree-org/iree/pull/19294/commits/8052d3f5dbc0076e1179add9d6db762a06dfd4d9

(I think you probably left the comment before my commit, or you did not refresh the old web page.)

Reply:
oh, i guess i clicked one of the commits, thinking that other commits were from a baseline PR.